src/screens/nano/NanoContractSelectBlueprint.js

```diff
@@ -0,0 +1,186 @@
+          <button type="button" className="mt-3 btn btn-hathor" onClick={onConfirmBlueprintSelection}>{t`Confirm`}</button>
+        </div>
+      );
+    }
+  }
+
+  const renderBlueprintAttributes = () => {
+    return (
+      <div className="table-responsive">
+        <table className="table table-striped table-bordered" id="attributes-table">
+          <thead>
+            <tr>
+              <th className="d-lg-table-cell">{t`Name`}</th>
+              <th className="d-lg-table-cell">{t`Type`}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {renderAttributes()}
+          </tbody>
+        </table>
+      </div>
+    );
+  }
+
+  const renderAttributes = () => {
+    return Object.entries(blueprintInformation.attributes).map(([name, type]) => {
+      return (
+        <tr key={name}>
+          <td>{name}</td>
+          <td>{type}</td>
+        </tr>
+      );
+    });
+  }
+
+  const renderBlueprintMethods = (key, header) => {
+    return (
+      <div className="table-responsive mt-5">
+        <table className="table table-striped table-bordered" id={`methods-table-${key}`}>
+          <thead>
+            <tr>
+              <th className="d-lg-table-cell">{header}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {renderMethods(key)}
+          </tbody>
+        </table>
+      </div>
+    );
+  }
+
+  const renderMethods = (key) => {
+    return Object.entries(blueprintInformation[key]).map(([name, detail]) => {
+      return (
+        <tr key={name}>
+          <td>{renderMethodDetails(name, detail.args, detail.return_type)}</td>
+        </tr>
+      );
+    });
+  }
+
+  const renderMethodDetails = (name, args, returnType) => {
+    const parameters = args.map(arg =>
+      `${arg.name}: ${arg.type}`
+    );
+    return `${name}(${parameters.join(', ')}): ${returnType === 'null' ? 'None' : returnType}`;
+  }
+
+  const renderBlueprintInformation = () => {
+    const blueprintId = blueprintIdRef.current.value;
+    return (
+      <div>
+        <p><strong>{t`ID: `}</strong>{blueprintId}</p>
+        <p><strong>{t`Name: `}</strong>{blueprintInformation.name}</p>
+        <h4 className="mt-5 mb-4">{t`Attributes`}</h4>
+        { renderBlueprintAttributes() }
+        { renderBlueprintMethods('public_methods', t`Public Methods`) }
+        { renderBlueprintMethods('private_methods', t`Private Methods`) }
+      </div>
+    );
+  }
+
+  return (
+    <div className="content-wrapper">
+      <BackButton />
+      <h4 className="my-4">{t`Select Blueprint`}</h4>
+      <div>
+        <form ref={formSelectBlueprintRef} id="formSelectBlueprint">
+          <div className="row">
+            <div className="form-group col-6">
+              <label>{t`Blueprint ID`}</label>
+              <input required autoFocus ref={blueprintIdRef} type="text" className="form-control" disabled={blueprintInformation !== null} />
```

Comment:
suggestion(non-blocking): Move the "Select new blueprint" button to here, so that we can clear the selected blueprint without scrolling all the way to the bottom of the screen.

Reply:
I agree it's better, see the screen changed

<img width="1312" alt="image" src="https://github.com/HathorNetwork/hathor-wallet/assets/3298774/d501f9ff-4121-47cb-b76c-4d1c10da4273">